src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -19,7 +19,7 @@ import kubernetesClusterExplorerControlPlane from 'images/kubernetes_screenshot-
 [New Relic](/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure) provides [control plane](https://kubernetes.io/docs/concepts/overview/components/#control-plane-components) support for your Kubernetes integration, allowing you to monitor and collect metrics from your cluster's control plane components. That data can then be found in New Relic and used to [create queries and charts](/docs/using-new-relic/data/understand-data/query-new-relic-data).
 
 <Callout variant="tip">
-  Unless otherwise specified, this page refers to the [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3). Details on how to configure control plane monitoring for v2 can be found in a specific section below.
+  Unless otherwise specified, this page refers to the [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3). Details on how to configure control plane monitoring for v2 can be found in a specific section below.
```

Comment:
I would change "can be found in a specific section below" to the actual section and section title

Reply:
Good catch!! Thank you!!